lambda_function_url_checker/README.md

```diff
@@ -2,21 +2,43 @@
 
 A generic-ish CloudWatch Synthetic Canary to health-check Lambda Function URLs using HTTP POST.
 
+## Overview
+
+This is a CloudWatch Synthetic Canary application that is Javasctip/Nodejs/Puppeteer-based. The following file structure is expected for an application like this:
```

Comment:
I might propose in the "Overview" section here, explaining what this canary does.  I realize the first line basicaslly explains this,

> A generic-ish CloudWatch Synthetic Canary to health-check Lambda Function URLs using HTTP POST.

.... but I still think it could be helpful.  Might be an opportunity to have language like,

> "This canary can be used to test any deployed Lambda that has a Function URL.  This is acheived by making an HTTP post request to the lambda's Function URL, with the URL and payload configured by env vars in that lambda's terraform project.  Broadly speaking, a 200 response is all that's required to demonstrate the lambda is not idle."

Again, maybe this is touched on elsewhere?  But I think it could be helpful in the first couple lines you encounter with the README.